src/components/Api.js

```diff
@@ -0,0 +1,90 @@
+export default class Api {
+  constructor(headers) {
+    // constructor body
+    this._headers = headers;
+  }
+
+  _checkResponse(res) {
+    if (res.ok) {
+      return res.json();
+    }
+    return Promise.reject(`Error: ${res.status}`);
+  }
+
+  getInitialCards() {
+    return fetch("https://around-api.en.tripleten-services.com/v1/cards", {
```

Comment:
According to the task you need to pass full `baseUrl` into the constructor of `Api`. See here https://snipboard.io/pRdaAZ.jpg . And in the code you need to use only: `this._baseUrl` (not to duplicate the url)